src/main/java/com/google/cloud/genomics/dataflow/readers/bam/Reader.java

```diff
@@ -180,9 +180,11 @@ void processRecord(SAMRecord record) {
   
   void dumpStats() {
     timer.stop();
+    long elapsed = timer.elapsed(TimeUnit.MILLISECONDS);
```

Comment:
This is not strictly related, but at one point I managed to break things in such a way that I got 0ms as the total time. This crashed, so I decided to fix it.